scripts/wait-for-app.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# 
+# Usage: pass in the host or host:port as the first argument
+#
+# This will then wait for a max of 5 mins for the application to start
+#
+
+RESOURCE=http://$1/assets/app.js
```

Comment:
add a `if [ "$#" -ne 1 ]; then echo "Wrong usage: ..."; exit 1; fi` (probably on multiple lines) in case someone doesn't pass the argument

Reply:
as per bash conventions the variables should all be lowercase unless they are environment variables passed from the outside

Reply:
Sorry I learnt my bash scripting skills in the mid-90's ... I'm not aware of any bash conventions - I can't seem to find any after searching :thinking:

Reply:
See https://github.com/koalaman/shellcheck/wiki/SC2154#rationale

Reply:
Done anyway :)